src/components/tools/Popper/stories/Popper.stories.tsx

```
@@ -0,0 +1,39 @@
+
+type Story = StoryObj<typeof Popper>;
+
+const meta: Meta<typeof Popper> = {
+    component: Popper,
+    title: 'tools/popper'
+};
+
+export default meta;
+
+export const Default: Story = {
+    render: ({ }) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)